realm/realm-library/src/androidTest/java/io/realm/TypeBasedNotificationsTests.java

```diff
@@ -502,38 +495,21 @@ public void onChange(Dog object) {
                 typebasedCommitInvocations.incrementAndGet();
                 assertEquals("Akamaru", dog.getName());
                 assertEquals(17, dog.getAge());
+                looperThread.testComplete();
             }
         });
 
-        realm.addChangeListener(new RealmChangeListener<Realm>() {
+        // Step 2: Trigger non-related commit
```

Comment:
They are still valid, but i did put a wrong `step 4` there. It should be `step 3`. will fix it.